Makefile

```diff
@@ -0,0 +1,2 @@
+test:
+	docker compose -f docker-compose.test.yaml up -d
```

Comment:
Instead of running a busy wait on the CI job, we could do this instead:
```
docker compose -f docker-compose.test.yaml up toxiproxy integration_test --abort-on-container-exit
```
this will also just run integration tests. WDYT?